cohorts/functions.py

```diff
@@ -84,7 +84,8 @@ def count_filter_fn(filterable_variant, **kwargs):
             return ((filterable_variant_function(filterable_variant) if filterable_variant_function is not None else True) and
                     filter_fn(filterable_variant, **kwargs))
         patient_id = row["patient_id"]
-        return cohort.load_variants(
+        return cohort.load_effects(
```

Comment:
Why this change vs. the existing `count_effects_function_builder` which does this?